core/src/main/java/net/glxn/qrgen/core/scheme/VCard.java

```diff
@@ -0,0 +1,177 @@
+package net.glxn.qrgen.core.scheme;
+
+import java.util.Map;
+
+/**
+ * A simple wrapper for vCard data to use with ZXing QR Code generator.
+ *
+ * See also http://zxing.appspot.com/generator/ and Contact Information
+ *
+ * @author Frederik Hahne <atomfrede@gmail.com>
+ */
+public class VCard extends AbstractQRCodeScheme {
+
+	private static final String BEGIN_VCARD = "BEGIN:VCARD";
+	private static final String NAME = "N";
+	private static final String COMPANY = "ORG";
+	private static final String TITLE = "TITLE";
+	private static final String PHONE = "TEL";
+	private static final String WEB = "URL";
+	private static final String EMAIL = "EMAIL";
+	private static final String ADDRESS = "ADR";
+	private static final String NOTE = "NOTE";
+
+	private String name;
+	private String company;
+	private String title;
+	private String phoneNumber;
+	private String email;
+	private String address;
+	private String website;
+	private String note;
+
+	public VCard() {
+	}
+
+	public VCard(final String vcardCode) {
+		if (vcardCode == null || !vcardCode.startsWith(BEGIN_VCARD)) {
```

Comment:
This breaks the existing VCARD impl. Is there a need for a constructor that overrides the vcardcode. If not I suggest keeping the name string as the default to ensure backwards compat.